apps/extension/src/popup.ts

```diff
@@ -1,10 +1,7 @@
 import { sessionExtStorage } from '@penumbra-zone/storage';
-import type {
```

Comment:
type import conversion seems to occur in many places

Reply:
Yep, will fix